engine/profiler/src/profiler_proc_utils.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020-2022 The Defold Foundation
+// Copyright 2014-2020 King
+// Copyright 2009-2014 Ragnar Svensson, Christian Murray
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+// 
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+// 
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#ifndef DM_PROFILER_PROC_PRIVATE_H
+#define DM_PROFILER_PROC_PRIVATE_H
+
+#include <script/script.h>
+
+#define SAMPLE_CPU_INTERVAL 0.25
```

Comment:
Is this used anyware outside of the .cpp file?
If not, move it to the .cpp file.